runtimes/manta/runtime/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ['Manta Network']
 edition = '2018'
 license = 'GPL V3'
 name = 'manta-runtime'
-version = '3.0.0'
+version = '3.0.1'
```

Comment:
Do we only bump runtime version?

Reply:
yeah, we should bump other version as well. good call.